rails/db/seeds.rb

```diff
@@ -1,10 +1,21 @@
-include FactoryBot::Syntax::Methods
+(1..10).each do |i|
+  email = "example-#{i}@test.com"
+  name = "example-#{i}"
 
-10.times do
-  user = create(:user)
+  user = User.find_or_create_by(email: email) do |u|
```

Comment:
[IMO]
ただのメモ書きなんで、無視してください

```suggestion
  role = i % 2

  user = User.find_or_create_by(email: email) do |u|
    u.name = name
    u.password = "password"
    u.password_confirmation = "password"
    u.role = role
  end
```

自分ならするかな